### PR TITLE
Move orders to Redux

### PR DESCRIPTION
**Pull request by chwallen**
Final PR of the three to handle our global state using Redux.

Removes a lot of duplicated code (such as websockets) and adds reusable order hooks.

### CODE REVIEW

**Comment by chwallen** [comment]
Rebased onto latest

File: backend/filters.py [hunk 1]
```diff
@@ -33,10 +33,10 @@ def get_younger_than_n_hours(self, queryset, field_name, value):
         return queryset.filter(created_timestamp__gte=time_threshold)
 
     def get_younger_than_datetime(self, queryset, field_name, value):
-        return queryset.filter(created_timestamp__gte=value)
+        return queryset.filter(created_timestamp__lte=value)
 
     def get_older_than_datetime(self, queryset, field_name, value):
-        return queryset.filter(created_timestamp__lte=value)
```
Comment (albinantti) [inline backend/filters.py:39]: Will switching these have any effects on the program?

**Review by albinantti — APPROVED** [review]

**Comment by chwallen** [comment]
Fixed a few bugs.

File: backend/filters.py (hunk 1 above)
Comment (chwallen) [reply backend/filters.py:39]: https://github.com/UTNkar/ordsys/pull/209/commits/f3f7a3e412e7bb4d542ef515518a56c5faa42bd6

**Comment by chwallen** [comment]
Rebased onto latest.

**Pull request merged**